src/functions/fakeroot_build.sh

```diff
@@ -26,7 +26,7 @@ fakeroot_build() {
         done
 
         field() {
-            cat "DEBIAN/control" | grep "${1}:" | awk -F": " '{print $2}'
+            cat "DEBIAN/control" | grep "${1}:" | cut -d":" -f 2
```

Comment:
As well as with the previous note (changing `:` to `: `), could you also add a space between `-d` and `"` like you did with the other files?

Reply:
sure sure will look into it and get this sorted.
Thanks a lot for testing and apologies for delays from my end, have been caught up at work lately.